charts/gateway/production-values.yaml

```diff
@@ -445,7 +471,43 @@ config:
       # changes will be required
       # Using Redis as a subChart is not recommended in production, please use your own Redis for this integration.
       enabled: false
-
+    additionalProviders: []
+    # - name: myRegionalRedisInstance1
+    #   redis:
+    #     tls:
+    #       existingSecret: myRegionalRedisCertSecret1
+    #       key: redis.crt
+    #       # path must match ssl.cert in your provider config
+    #       path: myRegionalRedisCert.crt
+    #   config: {}
+    # config:
```

Comment:
I think this line and the lines below need to have mre 2 spaces so that they're in the same level as "redis:" in line 476

Reply:
Updated